composer.json

```
@@ -3,14 +3,20 @@
     "license": "MIT",
     "type": "sulu-bundle",
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:alexander-schranz/sulu.git"
+        }
+    ],
```

This need to be removed when sulu/sulu is merged:

```suggestion
    "repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:alexander-schranz/sulu.git"
        }
    ],
```

@TheCadien the current Symfony 7 pull request requires this so we can test the compatibility correctly. We need to remove before merging it.